doc/nrf/known_issues.rst

```diff
@@ -831,6 +831,13 @@ SoftDevice Controller
 
 .. rst-class:: v1-5-0
 
+DRGN-15475: Samples using both the advertising and scanning but not connected state will fail to initialize the Bluetooth driver.
+  Samples using both the advertising and scanning but not connected state will fail to initialize the Bluetooth driver.
```

Comment:
The first line is the title for the known issue, the second is the description. So the first can be more general, while the second should be more descriptive. For example:
```suggestion
DRGN-15475: Samples might not initialize the Bluetooth driver correctly
  Samples that use both the advertising and the scanning state, but not the connected state, fail to initialize the Bluetooth driver.
  As a result, they report the following error/hang during startup/cannot initialize a connection/...?
```
Also, what do we mean with "Bluetooth driver"? Are we talking about the SoftDevice Controller? Or some driver in Zephyr?

Reply:
"SoftDevice controller HCI driver" Would be most descriptive I think.